boardgame-ui/src/utils/api/Game.js

```diff
@@ -67,12 +67,17 @@ export default class Game extends Model {
     return processedCollection;
   }
 
-  static async deleteGame(idGame) {
-    if(idGame == null) {
-      throw new Error('Cannot delete game with no ID.');
+  /**
+   * @override (uri not consistent for delete, fetch and update => need to override default behaviour)
+   */
+  async save() {
+    if(this.isNew()) {
+      return super.save();
+    }
+    else {
+      let {data} = await axios.put(`/event/${this.id_event}/${this.className}/${this.id}`, this.getPublicProperties());
```

Comment:
Note for the future, this should change when we'll handle games out of events.

Reply:
I think even now, we could remove the /event/:id/ part of the URL since the id of the game is sufficient to identify it. But it can wait :-)